.github/workflows/snapshot.yaml

```diff
@@ -0,0 +1,43 @@
+---
+
+name: Publish Snapshot
+
+on:
+  push:
+    branches:
+      - main
+      - '[0-9].x'
+  workflow_dispatch:
+
+jobs:
+
+  snapshot:
+    name: Deploy Snapshot
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up publishing to maven central
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            /var/cache/apt
+          key: lettuce-${{hashFiles('**/pom.xml')}}
+      - name: mvn offline
+        run: |
+          mvn -q dependency:go-offline
+      - name: deploy
+        run: |
+          mvn --no-transfer-progress \
+            -DskipTests deploy
```

Comment:
Out of curiosity: What is the rationale to skip the tests?

Reply:
Good question. Removed :)